### PR TITLE
chore: add logging to both client and server side

### PR DESCRIPTION
**Pull request by richardaum**
### Goal

Add logging to both client-side and server-side using `pino` library.

But why not `winston`? Because that library is not ready for browser usage.

Also,`NEXT_PUBLIC_LOG_CONTEXT` variable is being used to select which contexts you want to display. Right now, I have added `i18n` and `next.config.ts`, but others could be added.

This is you it is supposed to be used:

```bash
# if you want to display a single log context/scope
NEXT_PUBLIC_LOG_CONTEXT=i18n yarn run dev
# if you want to display multiple contexts
NEXT_PUBLIC_LOG_CONTEXT=i18n,next.config.ts yarn run dev
```

### CODE REVIEW

**Comment by vdfdev** [comment]
wow, sorry for the delay, I didnt see this PR! I will take a look tonight

**Review by vdfdev — APPROVED** [review]

File: web/package.json [hunk 1]
```diff
@@ -94,12 +94,14 @@
     "mobile-detect": "^1.4.4",
     "next": "9.5.5",
     "next-i18next": "^7.0.1",
+    "next-pino": "^0.0.1",
```
Comment (vdfdev) [inline web/package.json:97]: I think this dependency is unmaintained. Last commit was in 2018, and I got this warning:
```
warning " > next-pino@0.0.1" has incorrect peer dependency "pino@^4.13.0".
```

Seems to be working for now, but we might need to find a replacement.

**Review by vdfdev — CHANGES_REQUESTED** [review]
I tried to deploy this to staging and the bgio server is crashing with this error:
```
yarn run v1.22.5
$ node server/dist/server_bgio.js
(node:29) UnhandledPromiseRejectionWarning: Error: Missing `pino-pretty` module: `pino-pretty` must be installed separately
    at getPrettyStream (/appdata/node_modules/pino/lib/tools.js:188:13)
    at normalizeArgs (/appdata/node_modules/pino/lib/tools.js:352:16)
    at Object.pino [as default] (/appdata/node_modules/pino/pino.js:78:28)
    at Object.exports.createLogger (/appdata/server/dist/server_bgio.js:10388:32)
    at Object.<anonymous> (/appdata/server/dist/server_bgio.js:5471:24)
    at __webpack_require__ (/appdata/server/dist/server_bgio.js:20:30)
    at Object.<anonymous> (/appdata/server/dist/server_bgio.js:10353:15)
    at __webpack_require__ (/appdata/server/dist/server_bgio.js:20:30)
    at Object.<anonymous> (/appdata/server/dist/server_bgio.js:10335:14)
    at __webpack_require__ (/appdata/server/dist/server_bgio.js:20:30)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:29) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:29) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Done in 4.70s.
```

Let's fix this before merging

**Comment by richardaum** [comment]
While thinking a little more, I've decided to go back and not merging this PR as a simple console.log does the job for what I need.

**Pull request closed without merging**